challenges/objects-arrays.js

```diff
@@ -1,30 +1,53 @@
 // ==== Objects ====
 
-/* 
-  Given the following information about dinosaurs, create 3 objects: 
-  Use this pattern to create your objects: 
+/*
+  Given the following information about dinosaurs, create 3 objects:
+  Use this pattern to create your objects:
   object name, diet, weight, length, period
 */
 
+class Dinosaour {
```

Comment:
Pretty good implementation! Not necessary for this challenge, but I like that you went above here.